src/global.ts

```diff
@@ -118,6 +127,13 @@ declare global {
     };
   }
 
+  namespace Hooks {
+    interface HookConfig {
+        "lancer.statusesReady": () => boolean | void;
+        "lancer.statusInitComplete": () => boolean | void;
```

Comment:
There's some missing custom hooks but these were some obvious ones that needed to be added.